src/exporter/exporter.js

```diff
@@ -0,0 +1,101 @@
+const {fetchContentTypeDefinitions, fetchContentObjects} = require('../flotiq-api/flotiq-api');
+const fs = require('fs');
+
+exports.export = async (apiKey, directoryPath) => {
+
+    let contentTypedDefinitionsResponse = await getContentTypeDefinitionsData(apiKey)
+    let directoryNumber = 1;
+    let totalObjects = 0;
+    let totalPages = contentTypedDefinitionsResponse.total_pages;
+
+    let page = 1;
+    while (page <= totalPages) {
+        console.log(`CTD Page: ${page}/${totalPages}`);
+        for (let i = 0; i < contentTypedDefinitionsResponse.data.length; i++) {
+            let ctd = await clearCtd(contentTypedDefinitionsResponse.data[i]);
+            await saveSchema(ctd, directoryPath, directoryNumber);
+            let countSavedObjects = await saveObjects(apiKey, contentTypedDefinitionsResponse.data[i].name, directoryPath, directoryNumber);
+            totalObjects = totalObjects + countSavedObjects;
+            directoryNumber++;
+        }
+        page++;
+        if (page <= totalPages) {
+            contentTypedDefinitionsResponse = await getContentTypeDefinitionsData(apiKey, page);
+        }
+    }
+
+    let totalTypesDefinition = directoryNumber - 1
+    console.log('\x1b[32mSummary:');
+    console.log(`\x1b[32mTotal content types definitions: ${totalTypesDefinition}`);
+    console.log(`\x1b[32mTotal content objects: ${totalObjects}`);
```

Comment:
```suggestion
    console.log(`\x1b[32mTotal content objects: ${totalObjects}\u001b[0m`);
```
To prevent next commands styling, unset console colour

Before
![Desktop-screenshot (55)](https://user-images.githubusercontent.com/645716/152809903-6ccfe080-35b6-4c80-90c7-2fa651052c93.png)

After
![Desktop-screenshot (56)](https://user-images.githubusercontent.com/645716/152810139-fed1784c-cbe1-4578-843f-72591d3bb127.png)